doc/source/examples/multiphysics/static-irradiation/static-irradiation.rst

```diff
@@ -0,0 +1,471 @@
+|     :width: 620                                                                                                   |
+|     :name: Case setup                                                                                             |
+|                                                                                                                   |
++-------------------------------------------------------------------------------------------------------------------+
+
+The dimensions (:math:`H, \Delta h`, and :math:`L`) and the Dirichlet boundary condition values (:math:`\vec{u}_{\text{in}}, T_\text{in}`, and :math:`T_\text{0}`) are listed bellow.
+
++---------------------------+---------------------------+----------------------------+-----------------------------+
+| Parameter                 | Value                     | Parameter                  | Value                       |
++---------------------------+---------------------------+----------------------------+-----------------------------+
+| :math:`H`                 | :math:`170\;\mu\text{m}`  | :math:`\vec{u}_{\text{in}}`| :math:`0.1\;\text{m s}^{-1}`|
++---------------------------+---------------------------+----------------------------+-----------------------------+
+| :math:`\Delta h`          | :math:`20\;\mu\text{m}`   | :math:`T_{\text{in}}`      | :math:`298\;\text{K}`       |
++---------------------------+---------------------------+----------------------------+-----------------------------+
+| :math:`L`                 | :math:`600\;\mu\text{m}`  | :math:`T_{\text{0}}`       | :math:`298\;\text{K}`       |
++---------------------------+---------------------------+----------------------------+-----------------------------+
+
+There are three phases involved in this simulation: solid and liquid Ti6Al4V, and Argon. The metal-gas interface is handled by the VOF solver, while the solid-liquid interface is obtained from the temperature field of the HT solver. Hence, this example models a two-fluid problem: ``fluid 0`` corresponds to the Argon phase and ``fluid 1`` is the metal (solid and liquid), for which the solid part corresponds to a infinitely viscous fluid. 
+
+.. note::
+  To improve the performances of the solvers, all dimensional quantites in this example are based on the SI system except for the reference length, which is taken as :math:`1\;\text{mm}`.
+    
+--------------
+Parameter File
+--------------
+
+Simulation Control
+~~~~~~~~~~~~~~~~~~
+
+The time integration is handled by a 2nd-order backward differentiation scheme (``bdf1``) with a maximum time-step of :math:`\Delta t = 1.9 \times 10^{-8} \; \text{s} < \Delta t_\sigma` which corresponds to the capillary time-step constraint (see :doc:`capillary wave example <../capillary-wave/capillary-wave>`). We use adaptive time stepping with a maximum CFL of :math:`0.06` to prevent instability resulting from the explicit coupling between the NS and HT solvers through the recoil pressure and evaporative cooling. 
```

Comment:
```suggestion
The time integration is handled by a first order backward differentiation scheme (``bdf1``) with a maximum time-step of :math:`\Delta t = 1.9 \times 10^{-8} \; \text{s} < \Delta t_\sigma` which corresponds to the capillary time-step constraint (see :doc:`capillary wave example <../capillary-wave/capillary-wave>`). We use adaptive time stepping with a maximum CFL of :math:`0.06` to prevent instability resulting from the explicit coupling between the NS and HT solvers through the recoil pressure and evaporative cooling. 
```